packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -1,71 +1,69 @@
-                getLabelProps,
-                getMenuProps,
-                isOpen,
-                highlightedIndex,
-                inputValue,
-                getItemProps,
-            }) => (
-                <div className={`jkl-dropdown ${isOpen ? "jkl-dropdown--open" : ""} ${className}`} {...getInputProps()}>
-                    <button
-                        data-testid="jkl-dropdown-button"
-                        className="jkl-dropdown__toggler"
-                        {...getToggleButtonProps()}
-                    >
-                        {isOpen ? "Lukk" : "Åpne"}
-                    </button>
-                    <label
-                        className={`jkl-dropdown__label ${inputValue ? "jkl-dropdown__label--raised" : ""}`}
-                        {...getLabelProps()}
-                    >
-                        {label}
-                    </label>
-                    <input
-                        data-testid="jkl-dropdown-input"
-                        type="text"
-                        className={`jkl-dropdown__value ${inputValue ? "jkl-dropdown__value--show" : ""}`}
-                        value={inputValue !== null ? inputValue : undefined}
-                        readOnly
-                    />
-                    <ul className="jkl-dropdown__option" {...getMenuProps()}>
-                        {isOpen
-                            ? items.map((item, index) => (
-                                  <li
-                                      className={`jkl-dropdown__option--item ${
-                                          highlightedIndex === index ? "jkl-dropdown__option--item--highlighted" : ""
-                                      } ${inputValue === item ? "jkl-dropdown__option--item--selected" : ""}`}
-                                      {...getItemProps({
-                                          key: item,
-                                          index,
-                                          item,
-                                      })}
-                                  >
-                                      {item}
-                                  </li>
-                              ))
-                            : null}
-                    </ul>
-                </div>
-            )}
-        </Downshift>
-        /* eslint-enable jsx-a11y/label-has-associated-control, react/jsx-key */
+        <div className={className}>
+            <span className={`jkl-dropdown__label ${hasSelectedValue ? "jkl-dropdown__label--hover" : ""}`}>
+                {label}
+            </span>
+            <button className="jkl-dropdown__value">{hasSelectedValue ? selectedValue : "Velg"} </button>
+            <CoreToggle
+                className="jkl-dropdown__core-toggle"
+                popup={label}
+                hidden={hidden}
+                onToggle={() => setHidden(!hidden)}
+                onToggleSelect={(e: CoreToggleSelectEvent) => {
```

Comment:
This should go out as a named function. Though, it should not be solved here imho.

Reply:
Yeah, these first five comments should all be solved in #128, not here. It is definitely WIP, and @Saegrov has written a nice list of issues there that can be added to. For example, we don't accomodate an `onChange` function 😅